### DISCUSSION TITLE
Mail s being added to suppression list

### DISCUSSION THREAD

**Discussion by mattyg2301** [postalserver/postal#2814] [Help with using Postal]
Good morning all,

I have an issue which I am struggling to resolve, I've had a postal server running without issues for sometime now but all of a sudden certain sending get added to the suppression list with the below error:

FrozenError: can't modify frozen String

Not sure what that means or how to resolve, any assistance would be greatly appreciated.

**Comment by willpower232** [comment]
Are you definitely running the most recent version of Postal?

**Reply by mattyg2301** [reply]
Thank you for your reply, much appreciated!

I ran an update a month or two ago so assumed I was on the latest release however I wasn't and upgrading again seems to have solved the problem.

Thanks again for your assistance!
